### DISCUSSION TITLE
Optimal onboarding process?

### DISCUSSION THREAD

**Discussion by scottveirs** [orcasound/.github#14] [Ideas]
What should be on our Github org landing page? 

How much detail?

Pretty picture or complex schematic?

Community guidelines before onboarding links?

**Comment by UXBrendan** [comment]
@scottveirs 

I've put together an onboarding process for the UX Team, which includes that new volunteers read the GitHub org landing page. I'll collect feedback from newly onboarded volunteers regarding the content of the GitHub org landing page. 

This is the  high level onboarding process for the UX Team:
1. General onboarding letter sent to all new DLab applicants
2. The general onboarding letter direct them to the GitHub org landing page
3. Ideally, the new UX volunteer will read the content in the landing page and select the "[Onboarding Resources in our Product wiki](https://github.com/orcasound/product/wiki)" link under the "UX Team" heading
4. The Product wiki landing page has a link to an onboarding page
5. Onboarding UX volunteers will review the onboarding page wiki, which is a list of to do items and links to further wiki pages with resources

Thoughts on this process welcome!

Barring user feedback, I like the amount of content on the GitHub org landing page, and the image is helpful. There may be a different image we could use that shows a larger orca, but the current image does a good job of communicating our desire to help mitigate human interference of the SRKWs.
